src/downzip.js

```diff
@@ -9,9 +9,15 @@ const KEEPALIVE_INTERVAL_MS = 5000
 class DownZip {
     constructor(){
         this.worker = null
+    }
+
+    async register(options = {}) {
+        // Allow passing mapScriptUrl to service-worker-loader
+        const defaultMapScriptUrl = scriptUrl => scriptUrl
+        const mapScriptUrl = options.mapScriptUrl || defaultMapScriptUrl
 
         // Register service worker and let it intercept our scope
-        registerServiceWorker({
+        await registerServiceWorker(mapScriptUrl, {

```

Comment:
Added an options param. Now `mapScriptUrl` can be passed to `service-worker-loader`